docs/shared-content-source/docs/modules/develop/pages/cloudflow-configuration.adoc

```diff
@@ -149,6 +149,54 @@ The pod is limited to use `4Gi` of memory.
 It also shows how to set options to the JVM through an environment variable
 and how to add labels to a pod. 
 
+It also possible to add `volumes` to pods where the source of the volume is a secret. Then these can be mounted into the pod with `volume-mounts`. 
+
+There is an exception for `volume-mounts` regards being abel to set and specific type of pod. Will cover that later on, now here below, a sample of how to configure two `volumes` with secrets and how to mount them.  
+
+[source, hocon]
+----
+cloudflow.streamlets.my-streamlet {
+  kubernetes.pods.pod {
+   volumes {
+      foo {
+        secret {
+          name = mysecret
+        }
+      }
+      bar {
+        secret {
+          name = mysecret2
+        }
+      }
+    }
+    containers.container {
+      volume-mounts {
+        foo {
```

Comment:
I think we need to be a bit more specific that a name under volume-mounts (the name of a volume-mount) must match a name under volumes  (the name of the volume).

Reply:
done